### DISCUSSION TITLE
Shortcuts and sort order

### DISCUSSION THREAD

**Discussion by moongoses** [TBog/TBLauncher#305] [General]
Hello.
I have few questions.

What are shortcuts and how to use them? Whenever I press the shortcut icon I end with empty list. Cannot find any option to add or edit shortcuts.

Whenever I use the tag button to filter list, the list gets sorted alphabetically. Any way to change the sort order or just to filter the current sort order with the tagged entries only?

Is there a complete documentation explaining all the options? Most are self-explainatory, but there are some that I have no idea what they mean,

**Comment by TBog** [comment]
> Whenever I use the tag button to filter list, the list gets sorted alphabetically. Any way to change the sort order or just to filter the current sort order with the tagged entries only?

There are only 3 filters (apps, contacts, shortcuts)
What I mean is that you can't filter using a tag. You can only show all entries that have that tag.

There is a PR that would allow you to match multiple words, effectively filtering using name and tags, but it's not merged yet.

Could you please explain why you need this functionality? Maybe we can find a workaround.

**Reply by moongoses** [reply]
> There are only 3 filters (apps, contacts, shortcuts) What I mean is that you can't filter using a tag. You can only show all entries that have that tag.
> Could you please explain why you need this functionality? Maybe we can find a workaround.

Thx for prompt reply.

Sorry for misinterpretation of terminology. Under filtering I also understand showing the entries with same tag.

When I open the launcher, the list is sorted by recent used files with the most recent at the bottom. If I use some tag to display tagged entries only, the result list with all items having this tag is displayed in alphabetical order, not in the recently used order.

So if there are entries in the first list which have this tag, after activating the tag they can change their order to fit alphabetical sorting. What I want is to keep the same sort order (recent first) after choosing the tag. That is the way KISS works. 
So I was wandering if there is a setting for this behavior, because I couldn't find one.

> Shortcuts are created by apps you install and displayed by the launcher.
> 
> For example in Chrome you can press `Add to Home screen` and it will add a shortcut with the web page you are in.
> 
> Shortcuts are managed by the Android OS.


I understand now, thanks for clarification.

**Comment by TBog** [comment]
> Whenever I press the shortcut icon I end with empty list. Cannot find any option to add or edit shortcuts.

Shortcuts are created by apps you install and displayed by the launcher.

For example in Chrome you can press `Add to Home screen` and it will add a shortcut with the web page you are in.

Shortcuts are managed by the Android OS.

**Comment by TBog** [comment]
> Is there a complete documentation explaining all the options?

There is not.
Maybe you can start by creating one?
